Sources/PackageManager/SubsPackage.php

```diff
@@ -342,7 +342,7 @@ public static function read_tgz_data($data, $destination, $single_file = false,
 	 * @param array $files_to_extract
 	 * @return mixed If destination is null, return a short array of a few file details optionally delimited by $files_to_extract. If $single_file is true, return contents of a file as a string; false otherwise
 	 */
-	public static function read_zip_data($data, $destination, $single_file = false, $overwrite = false, $files_to_extract = null)
+	public static function read_zip_data(string $data, string $destination, bool $single_file = false, bool $overwrite = false, array$files_to_extract = null): mixed
```

Comment:
There is a typo in this signature. Need a space between the array type and $files_to_extract